src/genie/libs/parser/iosxe/tests/ShowRedundancy/cli/equal/golden_output_cat4500_expected.py

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Should say `expected_output = {` see https://github.com/CiscoTestAutomation/getting-started/blob/51d12c7e7ecc8ea1ce2a1f99e12a60903c70ab12/develop-guide/writeparser/writeparser.rst#test-your-new-parser for instructions.

Reply:
Perfect, It's already solved it, thanks